### PR TITLE
Fix JSDoc and regenerate scaffolding.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

**Comment by jmdobry** [comment]
Blocked by #38 (the samples tests will fail until #38 is fixed)

File: CONTRIBUTORS [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #
 Ace Nassri <anassri@google.com>
 Alexander Fenster <fenster@google.com>
+Alexander Fenster <github@fenster.name>
```
Comment (alexander-fenster) [inline CONTRIBUTORS]: I guess I'm not listed in .mailmap, hence the duplication.

Comment (jmdobry) [reply CONTRIBUTORS]: Feel free to update the `.mailmap` file throughout the `nodejs-*` repos and re-run `npm run generate-scaffolding` as desired.

**Comment by alexander-fenster** [comment]
Waiting for #38, OK to merge after that.

**Pull request merged**